Resolver/FormAnnotationResolver.php

```diff
@@ -159,12 +159,12 @@ protected function getBuiltObject(
         $entity = $request->attributes->get($annotation->getEntity());
 
         /**
-         * Creates form object from type
+         * Creates form object from type.
          */
-        $form = $formFactory->create($type, $entity);
+        $form = $formFactory->create(get_class($type), $entity);
```

Comment:
I'm just reviewing tests in travis and this should be finished very soon (maximum 15 minutes).
I'll ping you :)